src/Tizen.NUI.Components/Controls/Toast.cs

```diff
@@ -260,8 +246,6 @@ public Rectangle BackgroundImageBorder
         /// Gets or sets text left padding in toast.
         /// </summary>
         /// <since_tizen> 6 </since_tizen>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public int TextPaddingLeft
```

Comment:
Why don't you use Padding of View? Then we can remove the 4 properties(TextPaddingxxx)

Reply:
View padding is used in layout, and not satisfy our requirement, I think it's better to keep current.

Reply:
Done